### PR TITLE
refactor: create an interface for status console

### PR DESCRIPTION
**Pull request by TerryHowe**
**What this PR does / why we need it**:

Create a interface for console so that alternate implementations can exist. In doing this, I found the `Size()` method was not properly implemented, so I created a `GetHeightWidth()` method.

Partial: https://github.com/oras-project/oras/issues/1461

### CODE REVIEW

**Review by sabre1041 — APPROVED** [review]
LGTM

**Review by sabre1041 — APPROVED** [review]

File: cmd/oras/internal/display/status/console/console.go [hunk 1]
```diff
@@ -33,60 +33,75 @@ const (
...
-	c, err := console.ConsoleFromFile(f)
-	if err != nil {
-		return nil, err
-	}
-	return &Console{c}, nil
+// GetHeightWidth returns the width and height of the console.
+func (c *console) GetHeightWidth() (height, width int) {
+	windowSize, _ := c.Size()
+	return int(windowSize.Height), int(windowSize.Width)
 }
```
Comment (qweeah) [inline cmd/oras/internal/display/status/console/console.go:81]: Can we merge the implementation to those two functions? It's strange to make `Size` public but `console` is private.

**Review by qweeah — CHANGES_REQUESTED** [review]

**Review by qweeah — APPROVED** [review]
Thanks Terry for contributing. LGTM with minor sugggestions.

**Review by qweeah — APPROVED** [review]
LGTM with minor suggestions.

**Review by qweeah — APPROVED** [review]
LGTM

**Pull request merged**